src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/deliuscontext/CaseDetail.kt

```diff
@@ -40,7 +40,14 @@ data class Team(
   val borough: Borough?,
   val startDate: LocalDate?,
   val endDate: LocalDate?,
-)
+) {
+  fun toTeam(): uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.model.Team {
```

Comment:
could this maybe provide more context in the function name. Maybe toDomainEventTeam() (assuming that's what it is). I would say rename the target type but that'll probably just break something else as it's generated from open api :)